test/util.py

```diff
@@ -1,7 +1,10 @@
-from mycroft.configuration.config import LocalConf, DEFAULT_CONFIG
+from os.path import join, dirname
+from mycroft.configuration.config import LocalConf
 from copy import deepcopy
+from xdg import BaseDirectory
 
-__config = LocalConf(DEFAULT_CONFIG)
+__config = LocalConf(join(dirname(dirname(__file__)),
```

Comment:
Not a fan of this, but I'm not sure how else to resolve this without reintroducing `DEFAULT_CONFIG`...